### ISSUE TITLE
Remote connected: "Could not initialize SDL video"

### ISSUE THREAD

**Issue by alan0048** [Genymobile/scrcpy#4129]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 10 (pc) and Zorin OS (pc)
 - scrcpy version: 1.12.1
 - installation method: apt
 - device model: Motorola Moto G8
 - Android version: Android 11

Scrcpy works fine when I just do it from my Linux computer, but fails when I try the same thing through my remote pc.

**Working**: Zorin OS PC ---> Scrcpy ---> Phone

**Not Working**: Windows 10 (ssh) ---> Zorin OS PC ---> Scrcpy ---> Phone

**Warning message**:
_ERROR: Could not initialize SDL video: No available video device_

The whole point with scrcpy for me is remote accessing. Anyone got an idea of whats the problem? Why doesn't scrcpy work when It's used remotely?

**Comment by rom1v** [comment]
> Not Working: Windows 10 (ssh) ---> Zorin OS PC ---> Scrcpy ---> Phone

Of course, over ssh, by default you have no display. Even forwarding the display (`ssh -X`) would be a wrong solution, since you want to transfer the video stream, not the decoded frames in the window area.

For that, you should read [doc/tunnels](https://github.com/Genymobile/scrcpy/blob/master/doc/tunnels.md).

**Comment by alan0048** [comment]
Strange, how come scrcpy works with Raspberry Pi 4? On there you can SSH into the Raspberry Pi from another computer and stream whatever over SCRCPY. I was thinking this would be the same?

Or is this a feature ONLY limited to Raspberry PI?

**Comment by alan0048** [comment]
The way tunnels seems to work is required for someone to start the service on both ends. Won't work when you're alone. 

Guess I'll have to move onto another solution then.. Probably going Windows on my secondary computer and using some sort of screen mirror technique so that scrcpy may work remotely. Perhaps similar way exists on Linux (screen mirroring) since scrcpy doesn't work over ssh..

Getting SCRCPY over SSH to work would really be awesome.

**Comment by alan0048** [comment]
Strange tho. It does work over ssh when I did it for Nvidia Shield.

I would ssh into my remote router/network, then play scrcpy from my computer away from home. Thought this would be the exact same thing.
